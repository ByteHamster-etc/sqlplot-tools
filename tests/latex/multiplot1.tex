line1
% IMPORT-DATA test test.data
line2
%% MULTIPLOT(funcname) SELECT LOG(2,testsize) AS x, bandwidth AS y, MULTIPLOT
%% FROM test WHERE host='earth' ORDER BY MULTIPLOT,x
line3
%% MULTIPLOT(testsize) SELECT LOG(2,testsize) AS x, bandwidth AS y, MULTIPLOT
%% FROM test WHERE host='earth' ORDER BY MULTIPLOT,x
line4
