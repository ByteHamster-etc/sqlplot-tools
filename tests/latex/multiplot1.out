line1
% IMPORT-DATA test test.data
line2
%% MULTIPLOT(funcname) SELECT LOG(testsize) / LOG(2) AS x, bandwidth AS y, MULTIPLOT
%% FROM test WHERE host='earth' ORDER BY MULTIPLOT,x
\addplot coordinates { (10.1699,2.12566e+10) (11.0875,2.12068e+10) (11.6439,2.12568e+10) (12.0444,2.12565e+10) (12.6147,2.12549e+10) (13.0224,2.10335e+10) (13.5999,2.11052e+10) (14.0112,2.11423e+10) (14.3309,2.11649e+10) (14.5925,2.1178e+10) (14.8138,2.11881e+10) (15.0056,2.01671e+10) (15.3264,1.54346e+10) (15.5887,1.54457e+10) (16.0028,1.54706e+10) (16.5868,1.54763e+10) (17.0014,1.54782e+10) (17.5859,1.54527e+10) (18.0007,1.54039e+10) (18.5854,1.54242e+10) (19.0004,1.5424e+10) (19.5852,1.54265e+10) (20.0002,1.54166e+10) (20.3221,1.52702e+10) (20.5851,1.52696e+10) (20.8075,1.52691e+10) (21.0001,1.43805e+10) (21.17,1.43101e+10) (21.322,1.36672e+10) (21.4595,1.18576e+10) (21.585,1.09755e+10) (22,7.46265e+09) (22.322,5.23999e+09) (22.585,4.38139e+09) (22.8074,4.03316e+09) (23,3.84756e+09) (23.1699,3.74337e+09) (23.3219,3.66671e+09) (23.585,3.58156e+09) (23.8074,3.54338e+09) (24,3.52208e+09) (24.3219,3.50742e+09) (24.585,3.50313e+09) (24.8074,3.50257e+09) (25,3.50191e+09) (26,3.50248e+09) (27,3.50222e+09) (28,3.50208e+09) (29,3.50212e+09) (30,3.50154e+09) (31,3.50165e+09) (32,3.50154e+09) (33,3.50111e+09) (34,3.50009e+09) };
\addlegendentry{funcname=ScanRead64PtrUnrollLoop}
\addplot coordinates { (10.1699,2.00049e+10) (11.0875,2.00592e+10) (11.6439,2.01016e+10) (12.0444,2.00787e+10) (12.6147,2.01033e+10) (13.0224,1.98443e+10) (13.5999,1.99341e+10) (14.0112,1.99704e+10) (14.3309,1.99977e+10) (14.5925,2.00171e+10) (14.8138,2.00244e+10) (15.0056,1.95887e+10) (15.3264,1.33525e+10) (15.5887,1.33527e+10) (16.0028,1.33532e+10) (16.5868,1.33536e+10) (17.0014,1.33535e+10) (17.5859,1.33537e+10) (18.0007,1.33538e+10) (18.5854,1.33544e+10) (19.0004,1.33554e+10) (19.5852,1.33571e+10) (20.0002,1.33223e+10) (20.3221,1.25999e+10) (20.5851,1.25974e+10) (20.8075,1.25935e+10) (21.0001,1.18143e+10) (21.17,1.17322e+10) (21.322,1.11514e+10) (21.4595,9.63556e+09) (21.585,8.83049e+09) (22,5.70303e+09) (22.322,3.76486e+09) (22.585,2.97082e+09) (22.8074,2.62547e+09) (23,2.44873e+09) (23.1699,2.35262e+09) (23.3219,2.28865e+09) (23.585,2.22295e+09) (23.8074,2.19541e+09) (24,2.1806e+09) (24.3219,2.1704e+09) (24.585,2.16835e+09) (24.8074,2.16741e+09) (25,2.16724e+09) (26,2.16707e+09) (27,2.1674e+09) (28,2.16717e+09) (29,2.16721e+09) (30,2.16729e+09) (31,2.16717e+09) (32,2.16733e+09) (33,2.16723e+09) (34,2.1669e+09) };
\addlegendentry{funcname=ScanWrite64PtrUnrollLoop}
line3
%%% MULTIPLOT(testsize) SELECT LOG(testsize) / LOG(2) AS x, AVG(bandwidth) AS y, MULTIPLOT
% FROM test WHERE host='earth' GROUP BY MULTIPLOT ORDER BY MULTIPLOT,x
\addplot coordinates { (10.1699,2.06307e+10) };
\addlegendentry{testsize=1152}
\addplot coordinates { (11.0875,2.0633e+10) };
\addlegendentry{testsize=2176}
\addplot coordinates { (11.6439,2.06792e+10) };
\addlegendentry{testsize=3200}
\addplot coordinates { (12.0444,2.06676e+10) };
\addlegendentry{testsize=4224}
\addplot coordinates { (12.6147,2.06791e+10) };
\addlegendentry{testsize=6272}
\addplot coordinates { (13.0224,2.04389e+10) };
\addlegendentry{testsize=8320}
\addplot coordinates { (13.5999,2.05196e+10) };
\addlegendentry{testsize=12416}
\addplot coordinates { (14.0112,2.05563e+10) };
\addlegendentry{testsize=16512}
\addplot coordinates { (14.3309,2.05813e+10) };
\addlegendentry{testsize=20608}
\addplot coordinates { (14.5925,2.05975e+10) };
\addlegendentry{testsize=24704}
\addplot coordinates { (14.8138,2.06062e+10) };
\addlegendentry{testsize=28800}
\addplot coordinates { (15.0056,1.98779e+10) };
\addlegendentry{testsize=32896}
\addplot coordinates { (15.3264,1.43936e+10) };
\addlegendentry{testsize=41088}
\addplot coordinates { (15.5887,1.43992e+10) };
\addlegendentry{testsize=49280}
\addplot coordinates { (16.0028,1.44119e+10) };
\addlegendentry{testsize=65664}
\addplot coordinates { (16.5868,1.4415e+10) };
\addlegendentry{testsize=98432}
\addplot coordinates { (17.0014,1.44158e+10) };
\addlegendentry{testsize=131200}
\addplot coordinates { (17.5859,1.44032e+10) };
\addlegendentry{testsize=196736}
\addplot coordinates { (18.0007,1.43789e+10) };
\addlegendentry{testsize=262272}
\addplot coordinates { (18.5854,1.43893e+10) };
\addlegendentry{testsize=393344}
\addplot coordinates { (19.0004,1.43897e+10) };
\addlegendentry{testsize=524416}
\addplot coordinates { (19.5852,1.43918e+10) };
\addlegendentry{testsize=786560}
\addplot coordinates { (20.0002,1.43695e+10) };
\addlegendentry{testsize=1048704}
\addplot coordinates { (20.3221,1.39351e+10) };
\addlegendentry{testsize=1310848}
\addplot coordinates { (20.5851,1.39335e+10) };
\addlegendentry{testsize=1572992}
\addplot coordinates { (20.8075,1.39313e+10) };
\addlegendentry{testsize=1835136}
\addplot coordinates { (21.0001,1.30974e+10) };
\addlegendentry{testsize=2097280}
\addplot coordinates { (21.17,1.30211e+10) };
\addlegendentry{testsize=2359424}
\addplot coordinates { (21.322,1.24093e+10) };
\addlegendentry{testsize=2621568}
\addplot coordinates { (21.4595,1.07466e+10) };
\addlegendentry{testsize=2883712}
\addplot coordinates { (21.585,9.90297e+09) };
\addlegendentry{testsize=3145856}
\addplot coordinates { (22,6.58284e+09) };
\addlegendentry{testsize=4194432}
\addplot coordinates { (22.322,4.50242e+09) };
\addlegendentry{testsize=5243008}
\addplot coordinates { (22.585,3.6761e+09) };
\addlegendentry{testsize=6291584}
\addplot coordinates { (22.8074,3.32931e+09) };
\addlegendentry{testsize=7340160}
\addplot coordinates { (23,3.14814e+09) };
\addlegendentry{testsize=8388736}
\addplot coordinates { (23.1699,3.04799e+09) };
\addlegendentry{testsize=9437312}
\addplot coordinates { (23.3219,2.97768e+09) };
\addlegendentry{testsize=10485888}
\addplot coordinates { (23.585,2.90226e+09) };
\addlegendentry{testsize=12583040}
\addplot coordinates { (23.8074,2.8694e+09) };
\addlegendentry{testsize=14680192}
\addplot coordinates { (24,2.85134e+09) };
\addlegendentry{testsize=16777344}
\addplot coordinates { (24.3219,2.83891e+09) };
\addlegendentry{testsize=20971648}
\addplot coordinates { (24.585,2.83574e+09) };
\addlegendentry{testsize=25165952}
\addplot coordinates { (24.8074,2.83499e+09) };
\addlegendentry{testsize=29360256}
\addplot coordinates { (25,2.83457e+09) };
\addlegendentry{testsize=33554560}
\addplot coordinates { (26,2.83477e+09) };
\addlegendentry{testsize=67108992}
\addplot coordinates { (27,2.83481e+09) };
\addlegendentry{testsize=134217856}
\addplot coordinates { (28,2.83463e+09) };
\addlegendentry{testsize=268435584}
\addplot coordinates { (29,2.83467e+09) };
\addlegendentry{testsize=536871040}
\addplot coordinates { (30,2.83442e+09) };
\addlegendentry{testsize=1073741952}
\addplot coordinates { (31,2.83441e+09) };
\addlegendentry{testsize=2147483776}
\addplot coordinates { (32,2.83443e+09) };
\addlegendentry{testsize=4294967424}
\addplot coordinates { (33,2.83417e+09) };
\addlegendentry{testsize=8589934720}
\addplot coordinates { (34,2.83349e+09) };
\addlegendentry{testsize=17179869312}
this is the end
